src/core/createLogger.js

```diff
@@ -67,7 +67,7 @@ export default ({ getDebugEnabled, console, getMonitors, context }) => {
         data.parsedBody || data.body ? `response body:` : `no response body.`;
       log(
         "info",
-        `Request ${data.requestId}: Received response with status code ${data.status} and ${messagesSuffix}`,
+        `Request ${data.requestId}: Received response with status code ${data.statusCode} and ${messagesSuffix}`,
```

Comment:
You'll see `status` renamed to `statusCode` in a few places. That's because the object we were returning from `sendBeaconRequest`, `sendFetchRequest`, and `sendXhrRequest` previously had a `status` field on it, while the object returned from `sendNetworkRequest` had a `statusCode` field on it. I just wanted to make the names consistent.